src/modules/integrator-guide/pages/automation/automation-scripts/index.adoc

```diff
@@ -143,7 +143,7 @@ See xref:automation/automation-scripts/samples/server-scripts/index.adoc[server
 
 === Execution arguments
 
-Execution arguments (the `args` parameter of the <<automation-script-signature,`exec` function>>) contain the core data that your logic is able to work with.
+Execution arguments (the `args` parameter of the [#automation-script-signature-execution-arguments]#<<automation-script-signature-execution-arguments,`exec` function>>#) contain the core data that your logic is able to work with.
```

Comment:
Remove the `<<...>>` stuff from the original and just keep `... exec function.. `.
The link should point to the section that describes it but I refactored it out a little bit ago.